README.md

```diff
@@ -82,7 +82,7 @@ There are two main categories of samples: **Camera** and **Applications**. The s
 ## Instructions
 
 1. [**Install Zivid Software**](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/59080712/Zivid+Software+Installation).
-Note: The samples require Zivid SDK v2 (minor version 2.2 or newer).
+Note: The samples require Zivid SDK v2 (minor version 2.3 or newer).
```

Comment:
This change is not needed.

Reply:
True in theory, but it will fail on configuration because CMakeLists states 2.3.0

Reply:
Add a sentence to clarify then, see https://github.com/zivid/zivid-cpp-samples/pull/158#discussion_r615675488

Reply:
I'm wrong. This seems to work as a "minimum" version match